core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -587,27 +587,28 @@ class HttpGoogleServicesDAO(
 
     //First, get an access token to act as the Google APIs Service Agent that Deployment Manager runs as.
     val tokenRequestBody = new GenerateAccessTokenRequest().setScope(List(ComputeScopes.CLOUD_PLATFORM).asJava)
+    val saResourceName = s"projects/-/serviceAccounts/$billingProbeEmail" //the dash is required; a project name will not work. https://bit.ly/2EXrXnj
```

Comment:
some comments please about how this works and what security setup is required.
is there a reason we are acting as one SA to get creds for another instead of just having the private key?

Reply:
can do. honestly i did this because adding more keys to vault for 6 different envs is a boring chore. and it makes for more boring chores down the road when we have to refresh the keys